site/app/templates/admin/Configuration.twig

```diff
@@ -30,6 +30,9 @@
             <label for="forum-enabled">
                 <div class="option-title">Enable Discussion Forum</div>
                 <div class="option-alt">Choose whether to enable a forum for this course.</div>
+                {% if categories_empty %}
+                    <h4 id="forum-category-warning" class="red-message">Categories must be created in order to use the Discussion Forum. Click the link below to add categories.</h4>
+                {% endif %}
```

Comment:
Why isn't the bottom link also part of this twig conditional that way we wouldn't need a client side check to hide it initially

Reply:
No reason I can think of besides it being there originally, I will add it to the twig conditional.